tests/kernel/early_sleep/src/main.c

```diff
@@ -91,7 +91,7 @@ SYS_INIT(test_early_sleep_app, APPLICATION, CONFIG_KERNEL_INIT_PRIORITY_DEVICE);
  *
  * @see k_sleep()
  */
-static void test_early_sleep(void)
+ZTEST(earlysleep_1cpu, test_early_sleep)
```

Comment:
no need for 1cpu in the suite name.

Reply:
You mean rename the suite name if it only contains cases of 1cpu?
We also have combo scenario that both have `ztest_1cpu_unit_test` and `ztest_unit_test`, like kernel/context. See #47229 ,is it ok to change like that PR?